src/breadcrumbs/breadcrumbs.tsx

```diff
@@ -22,53 +21,56 @@ const ThemelessBreadcrumbs = React.forwardRef<
       children,
       showTrailingSeparator = false,
       overrides,
-
       size = 'medium',
       ...rest
     },
     ref,
   ) => {
     const theme = useTheme();
-    const separatorOverrides = {
-      ...theme.componentDefaults.breadcrumbSeparator,
-      ...filterOutFalsyProperties(overrides?.separator),
-    };
-
-    const iconToken = getToken(
-      {theme, overrides},
-      `breadcrumbSeparator.${size}`,
-      `${size}`,
-      'iconSize',
-    );
-
-    const iconStylePresetToken = getToken(
-      {theme, overrides},
-      `breadcrumbSeparator.${size}`,
-      `${size}`,
-      'stylePreset',
-    );
-    const iconSpaceToken = getToken(
-      {theme, overrides},
-      `breadcrumbSeparator.${size}`,
-      `${size}`,
-      'paddingInline',
-    );
 
     const [BreadcrumbsIcon, BreadcrumbsIconProps] = getComponentOverrides(
```

Comment:
only components start with UpperCase
```suggestion
    const [BreadcrumbsIcon, breadcrumbsIconProps] = getComponentOverrides(
```